config/database.php

```diff
@@ -63,6 +63,20 @@
             'engine' => 'InnoDB ROW_FORMAT=DYNAMIC',
         ] + $db_config,
 
+        'mysql_testing' => [
```

Comment:
That is not the right reason. Some people also use a different db instead of connection. You need to tell me why this approach is better. I think using a different db name is better because everybody has their DB set up nicely already. Just change the DB name and everything will work fine. It's just a single variable change. This approach unnecessarily adds a lot of lines of code just to establish a connection that is the same as the development.